nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -656,7 +656,7 @@ def add_electrical_series(
             eseries_kwargs.update(conversion=1e-6)
             eseries_kwargs.update(channel_conversion=channel_conversion)
 
-    trace_dtype = recording.get_traces(channel_ids=[0], end_frame=1).dtype
+    trace_dtype = recording.get_traces(channel_ids=channel_ids[:1], end_frame=1).dtype
```

Comment:
```suggestion
    trace_dtype = recording.get_traces(channel_ids=channel_ids[0], end_frame=1).dtype
```

You're right, there's no guarantee that the actual index `0` is in the `channel_ids` list. Should just query the zeroth index of the known `channel_ids` list instead.